frontend/src/app/problems/add/problem-meta-data.tsx

```
@@ -1,217 +1,580 @@
+                        No {resource}s found
+                    </Typography>
+                </Box>
+            </Grid>
+            <Grid size={6}>
+                {getNewResourceLink(resource)}
+            </Grid>
+        </Grid>
+    )
+}
+
+function getFooterOption(resource: string, allowCreate: boolean = false, onClick?: () => void): ReactNode {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)